resources/devices/mobile-phone/htc.json

```diff
@@ -974,32 +974,6 @@
             "isTablet": "false"
         }
     },
-    "HTC HD mini": {
```

Comment:
These two are essentially copies of eachother AFAICT, and I moved them to "HTC T555" to be consistent with other HTC devices.